provider/cloudflare.go

```diff
@@ -150,6 +155,26 @@ func (p *CloudFlareProvider) Zones(ctx context.Context) ([]cloudflare.Zone, erro
 	result := []cloudflare.Zone{}
 	p.PaginationOptions.Page = 1
 
+	// if there is a zoneIDfilter configured
```

Comment:
I don't think this is needed, the code already explains it so I would drop the comments

Reply:
(note: I didn't write this: the author of  #1307 did):

The following line that notes that empty strings are used by tests is informative.

Reply:
Ah I see my bad, I would still like to drop those lines. Maybe only keep the second ... blank string for unit tests ...